### ISSUE TITLE
bug - Mat-chip-list - has incorrect aria role when empty

### ISSUE THREAD

**Issue by chrisbubernak** [angular/material2#7564]
#### Bug, feature request, or proposal:

Bug: elements with "listbox" role require children "option" role but an empty mat-chip-list is still a "listbox".

#### What is the expected behavior?

Not sure what the best solution is but my guess would be that mat-chip-list should have a different role when it has no children. Normally the mat-chip-list should have chips but there could be situations where the mat-chip-list starts empty and has chips dynamically added.

#### What is the current behavior?

Currently mat-chip-list's role is always "listbox".

#### What are the steps to reproduce?
Providing a StackBlitz/Plunker (or similar) is the *best* way to get the team to see your issue. <br/>
Plunker starter (using on `@master`): https://goo.gl/DlHd6U<br/>
StackBlitz starter (using latest `npm` release): https://goo.gl/wwnhMV<br/>
